### PR TITLE
Minor documentation and script cleanups

### PR DESCRIPTION
**Pull request by a1346054**
Cleans up several minor issues in the codebase.

Everything was tested to not break by running the changed parts.

- ✅ Bug fix (non-breaking change that fixes an issue)
- ✅ Documentation (non-code change)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Before submitting a bug report, check if the problem has already been reported.
 
 ### Discuss with the team
 
-As with feature requests, you can talk to the KeePassXC team about bugs, new features, other issues and pull requests on the dedicated issue tracker, or in the IRC channel on Freenode (`#keepassxc-dev` on `irc.freenode.net`, or use a [webchat link](https://webchat.freenode.net/?channels=%23keepassxc-dev)).
+As with feature requests, you can talk to the KeePassXC team about bugs, new features, other issues and pull requests on the dedicated issue tracker, or in the IRC channel on Libera.Chat (`#keepassxc-dev` on `irc.libera.chat`, or use a [webchat link](https://web.libera.chat/#keepassxc-dev)).
```
Comment (louib) [inline .github/CONTRIBUTING.md]: @a1346054 this should actually link to the matrix channels. See the badges in the README for details.

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Detailed instructions are available in the [Build and Install](./INSTALL.md) pag
 
 ## Contributing
 
-We are always looking for suggestions on how to improve KeePassXC. If you find any bugs or have an idea for a new feature, please let us know by opening a report in the [issue tracker](https://github.com/keepassxreboot/keepassxc/issues) on GitHub or join us on IRC in [freenode](https://webchat.freenode.net/) channels #keepassxc and #keepassxc-dev.
+We are always looking for suggestions on how to improve KeePassXC. If you find any bugs or have an idea for a new feature, please let us know by opening a report in the [issue tracker](https://github.com/keepassxreboot/keepassxc/issues) on GitHub or join us on IRC in [Libera.Chat](https://web.libera.chat/) channels #keepassxc and #keepassxc-dev.
```
Comment (louib) [inline README.md]: @a1346054 this should also link to the matrix channels. See the badges for details.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (droidmonkey) [reply .github/CONTRIBUTING.md]: Agree or removed completely since we have the badges at the top of readme already.

Comment (a1346054) [reply .github/CONTRIBUTING.md]: Is that instead of mention of IRC, or in addition to it? I have no experience with matrix, so don't know how to link to it.

File: release-tool [hunk 3]
```diff
@@ -61,7 +61,7 @@ printUsage() {
         cmd="COMMAND"
     fi
 
-    printf "\e[1mUsage:\e[0m $(basename $0) $cmd [options]\n"
+    printf "\e[1mUsage:\e[0m $(basename "$0") $cmd [options]\n"
```
Comment (droidmonkey) [inline release-tool:64]: This is invalid, you break the double quotes

File: share/translations/update.sh [hunk 4]
```diff
@@ -16,23 +16,23 @@
...
     UPDATE=false
-elif [ "$1" == "update" ]; then
+elif [[ $1 == "update" ]]; then
     PUSH=false
     PULL=false
-elif [ "$1" != "" ]; then
+elif [[ $1 != "" ]]; then
     echo "Unknown command '${1}'"
-    echo "Usage: $(basename $0) [update|pull|push] [additional tx options]"
+    echo "Usage: $(basename "$0") [update|pull|push] [additional tx options]"
```
Comment (droidmonkey) [inline share/translations/update.sh:35]: You break the double quotes

File: utils/makeappicons.sh [hunk 5]
```diff
@@ -148,27 +148,27 @@ map() {
...
     SRC="$MATERIAL/svg/$MAT.svg"
-    if [ ! -f "$SRC" ];then
+    if [[ ! -f $SRC ]]; then
         echo -e "${RED}Error: Source for ${NAME} doesn't exist: ${SRC}${NC}"
         continue
     fi
 
     # Replace the icon file with the source file
-    cp "$SRC" "$DST" || exit
+    cp -- "$SRC" "$DST" || exit
```
Comment (droidmonkey) [inline utils/makeappicons.sh:171]: Why this change?

File: utils/makedbicons.sh [hunk 6]
```diff
@@ -39,15 +39,15 @@ fi
...
+    if [[ ! -f $SRC ]]; then
         echo -e "${RED}Cannot find source icon for ${2} (${SRC})${NC}"
         return
     fi
 
     # Copy the source file to the destination, keeping
     # the source file's extension
     DST="$DSTDIR/${2}.svg"
-    cp $SRC $DST
+    cp -- "$SRC" "$DST"
```
Comment (droidmonkey) [inline utils/makedbicons.sh:50]: Again, why this change?

**Review by droidmonkey — CHANGES_REQUESTED** [review]

File: utils/makeappicons.sh (hunk 5 above)
Comment (a1346054) [reply utils/makeappicons.sh:171]: The `--` is there because $SRC cannot be guaranteed to *not* start with a `-` and hence could be interpreted as an option to `cp`. Using `--` means 'end processing of option arguments'. It is likely an unnecessary and overly cautious change, sure, I can rebase without it.

Taken from Guideline 10 at https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_02

File: utils/makedbicons.sh (hunk 6 above)
Comment (a1346054) [reply utils/makedbicons.sh:50]: The `--` is just like in the earlier comment, and the ``""`` around $SRC and $DST are to prevent the usual issues with accidental globbing and field splitting.

File: utils/makeappicons.sh (hunk 5 above)
Comment (droidmonkey) [reply utils/makeappicons.sh:171]: Fair enough makes sense to me now

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (a1346054) [reply .github/CONTRIBUTING.md]: Added mention of matrix now.

File: README.md (hunk 2 above)
Comment (a1346054) [reply README.md]: Added mention of matrix now.

**Review by louib — APPROVED** [review]

**Review by droidmonkey — APPROVED** [review]
